packages/react/src/hooks/useRerender/useRerender.ts

```diff
@@ -2,8 +2,10 @@ import { useCallback, useReducer } from 'react'
 
 type Rerender = () => void
 const reducer = (state: number) => state + 1
-export const useRerender = (): Rerender => {
+const useRerender = (): Rerender => {
   const [, dispatch] = useReducer(reducer, 0)
 
   return useCallback(() => dispatch(), [])
 }
+
+export default useRerender
```

Comment:
I wonder why you used default export!
Any advantage?!😃

Reply:
I just unified with other hook.

Do you prefer named exports? (Actually I am)